pkg/resource/v1/certconfig/delete.go

```diff
@@ -2,14 +2,43 @@ package certconfig
 
 import (
 	"context"
+	"fmt"
 
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/operatorkit/framework"
 )
 
 func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange interface{}) error {
 	return nil
 }
 
+// NewDeletePatch is called upon observed custom object deletion. It receives
+// the deleted custom object, the current state as provided by GetCurrentState
+// and the desired state as provided by GetDesiredState. NewDeletePatch
+// analyses the current and desired state and returns the patch to be applied by
+// Create, Delete, and Update functions.
```

Comment:
Nitpick level max: Usual ordering would be Create, Update, Delete (CRUD and stuff). (Feel free to ignore as it's a really nitpicky nitpick)

Reply:
Sure, I'll fix this. :)